src/components/CartList.js

```diff
@@ -41,6 +42,15 @@ export const CartList = () => {
     updateCart(cart.filter((item) => item._id !== product._id));
```

Comment:
why not call this separately and then pass the returned data from `updateCart()` to the dispatch? This would make it clearer what the updateCart() returns.